modules/nixos/buildbot.nix

```diff
@@ -25,6 +25,16 @@ in
   sops.secrets.buildbot-nix-workers = { };
   sops.secrets.cachix-auth-token = { };
 
+  services.buildbot-nix.master.pullBased = {
+    pollInterval = 1800;
+    repositories = {
+      "astro/microvm.nix" = {
+        url = "https://github.com/qowoz/microvm.nix.git";
+        defaultBranch = "main";
+      };
```

Comment:
```suggestion
      "astro/microvm.nix" = {
        url = "https://github.com/qowoz/microvm.nix.git";
        defaultBranch = "main";
        evalWorkerCount = 4;
      };
```
@Mic92 Could we make evalWorkerCount configurable per repo? This repo uses a lot of memory and I really don't want to have to reduce it just for one repo.

Reply:
@zowoq sounds doable and useful. Please open an issue. We have most of the code needed for this to happen already there.